pom.xml

```diff
@@ -37,10 +44,8 @@
     <version.assertj>3.5.2</version.assertj>
     <version.jcip.annotations>1.0</version.jcip.annotations>
     <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
+    <version.snakeyaml>1.17</version.snakeyaml>
```

Comment:
yeah but we are excluding it if user is using chameleon-core. Look [here](https://github.com/arquillian/arquillian-container-chameleon/pull/68/files#diff-9e8ef6416199524dc8d332f82bc6f5cfR93)

It's added only for configuration module to remove resolver dependency